client/src/components/DataFiles/DataFilesBreadcrumbs/DataFilesBreadcrumbs.scss

```diff
@@ -34,3 +36,102 @@
     max-width: 700px;
   }
 }
+
+.data-files-btn {
+  background-color: var(--global-color-accent--normal);
+  border-color: var(--global-color-accent--normal);
+  border-radius: 0;
+  font-family: 'Roboto, sans-serif';
+}
+
+.data-files-btn-cancel {
+  border-radius: 0;
+}
+
+#path-button-wrapper {
+  padding-left: var(--horizontal-buffer);
+}
+#data-files-path {
+  font-size: 12px;
+  padding: 5px 10px;
+  border-color: #707070;
+}
+
+.data-files-nav {
+  padding-top: 20px;
+}
+
+/* HACK: Quick solution to prevent styles from cascading into header dropdown */
+.go-to-button-dropdown {
+  .dropdown-menu {
```

Comment:
Most of this looks like it is just copied straight from `DataFilesSidebar.scss`. Maybe even with some changes for efficiency. Are you comfortable taking this further by trying to share these styles in one location, and adapting as necessary for the two use cases?

Reply:
Imported DataFilesSidebar.scss directly into the DataFilesBreadcrumbs.scss file so that it can use styles, unless overwritten inside the DataFilesBreadcrumbs.scss file.

Reply:
This feels like a step of an incomplete process.

Architecturally, as is, this might be more troublesome than how you had it.

Issues:
- Such a CSS @import suggests "breadcrumbs styles use sidebar styles", which is misleading.
- CSS is still duplicated in the repo. And now sometimes rules are overwritten with exact duplicates.

Do you mind if I try a solution? It would be in a separate PR.

Reply:
This is the quickest way I found to share the majority of styles without duplication:
- #875

This was not intuitive. Other CSS oddities create a fog in front of the effort. I'll try to help with more in separate PRs.

Reply:
I tested your version and it looks good to me. Thanks Wesley! And thanks for moving the placement of the dropdown menu down too, it looks better now. 

I did unfortunately find a side effect just now (nothing you did) that I need to address. Because the DataFilesCopyModal also uses the DataFilesBreadcrumbs component, it's having unintended side effects with the navigation of the page outside the modal--the dropdown menu is also rendering in the copy modal and looks off and is not navigating in the modal correctly. So I'll need to rewrite DataFilesCopyModal, a little so that it doesn't mess up the copy modal for users.

Reply:
Same with DataFilesMoveModal

Reply:
Okay. I don't fully understand, but I trust you. Do you want to merge #875 into this PR and write atop it, or have me update after you refactor?

Reply:
_**Update**: I got PR approval and request to merge via private message._